ansible/deploy_backend_final.yml

```
@@ -0,0 +1,213 @@
+          After=network.target
+
+          [Service]
+          Type=simple
+          User=root
+          ExecStart=/usr/bin/java -Dspring.profiles.active=docker -jar {{ backend_dest }}/app.war
+          Restart=on-failure
+          RestartSec=30
+          WorkingDirectory={{ backend_dest }}
+          StandardOutput=journal
+          StandardError=journal
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)